### PR TITLE
New external command for scenareos where LBs are attached externally not explicitly via feed

### PR DESCRIPTION
**Pull request by chbatey**
The motivation for this is to run feed in GKE.

Firstly we were going to follow a similar model to the NLBs are attach to pre-existing NLBs but in the end it was simpler to allow the GKE ingress controller to create and attach internal/external LBs to the feed pods and feed update each ingress to point to a static host. This requires a service/ingress for feed.

So looks like this:

image

### CODE REVIEW

**Review by howardburgess — APPROVED** [review]
Elegant solution 👍🏻

**Review by jsravn — CHANGES_REQUESTED** [review]
I think this can be even more generic than "external" since it will work with any setup that has static domains for feed. So I suggest we rename it to "static" and update the README as well.

File: feed-ingress/cmd/external.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+	externalCmd.Flags().StringVar(&externalHostname, "external-hostname", "",
+		"Hostname for external ingress")
+	externalCmd.Flags().StringVar(&internalHostName, "internal-hostname", "",
+		"Hostname for internal ingress")
+
+}
+
+func empty(kubernetesClient k8s.Client, updaters []controller.Updater) ([]controller.Updater, error) {
+
```
Comment (jsravn) [inline feed-ingress/cmd/external.go:34]: remove whitespace

**Comment by chbatey** [comment]
Will re-name to static and update readme 👍

**Review by jsravn — APPROVED** [review]

**Pull request merged**